### PR TITLE
Update datasources.json with 32 new Institutions

### PR DESCRIPTION
**Pull request by LukePrior**
I have added these institutions and sorted them into the right spots. I used the data from: https://github.com/Uskompuf/Australian-Open-Banking-Data-Database

### CODE REVIEW

**Comment by CDR-API-Stream** [comment]
Thanks @LukePrior, apologies it has taken us some time to review this change.

All 32 additions have been reviewed and we will merge these into the Banking Product Comparator. 

For any organisation who has been included in the above list and would like to amend their entry - please reach out to contact@consumerdatastandards.gov.au

File: public/datasources.json [hunk 1]
```diff
@@ -1,70 +1,102 @@
...
+    {"name":"Unity Bank","url":"https://ibanking.unitybank.com.au/openbanking/cds-au/v1","icon":"https://asset.unitybank.com.au/images/Unity-logo-335x115.png"},
+    {"name":"Up","url":"https://api.up.com.au/cds-au/v1","icon":"https://up.com.au/assets/images/logo_400x400.png"},
+    {"name":"Virgin Money Australia","url":"https://secure.api.virginmoney.com.au/cds-au/v1","icon":"https://virginmoney.com.au/content/dam/virginmoney/logos/logo_footer.png"},
+    {"name":"Volt Bank","url":"https://api.voltbank.com.au/cds-au/v1","icon":"https://openbanking.voltbank.com.au/logos/logo?v=1600930338006"},
+    {"name":"Warwick Credit Union","url":"https://onlinebanking.wawcu.com.au/OpenBanking/cds-au/v1"},
+    {"name":"WAW Credit Union","url":"https://onlinebanking.wawcu.com.au/OpenBanking/cds-au/v1"},
+    {"name":"Woolworths Team Bank","url":"https://online.woolworthsteambank.com.au/OpenBanking/cds-au/v1"},
+    {"name":"Xinja Bank","url":"https://cdr.openbank.api.xinja.com/cds-au/v1","icon":"https://xinja.com.au/wp-content/uploads/2019/09/xinja-email-logo-grey-pink-400x96.png"}
```
Comment (vadkor) [inline public/datasources.json:101]: Please remove Xinja. They have discontinued their banking operations in Australia: https://www.itnews.com.au/news/neobank-xinja-hands-back-its-banking-licence-559059

**Review by vadkor — CHANGES_REQUESTED** [review]
Please resolve the conflicts. Removing the no longer required `/cds-au/v1` from each URL will probably take care of most of them.

**Comment by LukePrior** [comment]
I don't know how to add commits to this pull request my changes are here: https://github.com/LukePrior/banking-products-comparator. Do I just open a new request @vadkor ?

**Comment by LukePrior** [comment]
I opened https://github.com/ConsumerDataStandardsAustralia/banking-products-comparator/pull/145

**Comment by vadkor** [comment]
> I don't know how to add commits to this pull request my changes are here: https://github.com/LukePrior/banking-products-comparator. Do I just open a new request @vadkor ?

Normally you just add commits to the source branch of your PR and they appear here when pushed as usual. No worries. Will continue in #145.

**Pull request closed without merging**